### ISSUE TITLE
Navigator.scheme removed ?

### ISSUE THREAD

**Issue by nicoaldegheri** [devxoul/URLNavigator#85]
Hello,

I just try to update to your new 2.0 URLNavigator version and I can't find a way to set automatically my URL scheme in the Navigator like with `Navigator.scheme` in 1.0 version
I need to do this because I create two apps from the same base code but with different scheme.
And I have a lot of `Navigator.map("my/url/path/without/scheme")` so duplicate all of these with each scheme version isn't a good solution.

Did I missed something?

Thank!

**Comment by bill350** [comment]
@nicoaldegheri is this related to https://github.com/devxoul/URLNavigator/issues/58 ? 
Why it has been removed @devxoul ?

**Comment by nicoaldegheri** [comment]
Yes, it's related to #58 but it seems that between the two issues `Navigator.scheme` was added then removed.

**Comment by bill350** [comment]
Yes but I didn't understand because of the non-closed issue #58 🙂 Thanks.

**Comment by devxoul** [comment]
It is not related to #58. I totally forgot it when I was rewriting the whole code. I'll make a new release soon. :pray:
